### DISCUSSION TITLE
Image pull secret

### DISCUSSION THREAD

**Discussion by fabland** [keycloak/keycloak#11999] [Archive]
At the moment if I want to use and image from a registry that needs authentication I have to configure it through the unsupported property in the Keycloak Crd.

It would be good to have the imagePullSecret as a standard property of the Keycloak Crd as building a custom image seems to be quite a common case now e.g. to add a customized theme. Or maybe have an example in the docs too?

**Comment by andreaTP** [comment]
Makes sense to me to include `imagePullSecret` as a first-class field in the CR. cc. @vmuzikar

**Reply by vmuzikar** [reply]
@fabland Thanks for your interest in the new Operator!

👍  Created: https://github.com/keycloak/keycloak/issues/12094
